MekHQ/data/universe/factions.xml

```diff
@@ -1777,6 +1787,9 @@ tags - a comma-separated list of tags. Currently recognised tags: "is", "periphe
       <nameGenerator>General</nameGenerator>
       <colorRGB>191,171,120</colorRGB>
 		<tags>periphery,minor</tags>
+		 <start>2779</start>
+		 <!-- May be sooner than that, but no later -->
+		 <end>2822</end>

```

Comment:
FRR disappeared during the 2nd SW, which lasted 2830 - 2864. End date should fall in that range.

Reply:
I noticed that, but it seems that they had no planets by 2822. Probably a small inconsistency. I can make it 2830-ish.

Reply:
Trying to source a date, I found this in The First Succession War (p100), emphasis added:

> In a 2792 naval raid, the Commonwealth attempted to seize Finmark’s shipyard...the spiteful Lyran commander ordered a nuclear bombardment of the planetary surface, targeting all major settlements for destruction. With the death of its capital world and the associated political, industrial, and economic resources, *the FFR collapsed over the next decade*.

Reply:
Fair enough, I'm good with the date then.